decidim-admin/app/controllers/decidim/admin/moderations_controller.rb

```diff
@@ -67,6 +67,20 @@ def unhide
         end
       end
 
+      def bulk_action
+        Admin::BulkAction.call(current_user, params[:bulk_action], reportables) do
```

Comment:
```suggestion
        Admin::BulkAction.call(current_user, params[:bulk_action], moderations) do
```
related to the previous comment

Reply:
Done